docs/quickstarts/openshift-ai-rhods-fasttrack/openshift-ai-rhods-fasttrack.yml

```diff
@@ -2,16 +2,16 @@ apiVersion: console.openshift.io/v1
 kind: QuickStarts
 metadata:
   name: openshift-ai-rhods-fasttrack
-  externalDocumentation: true
+  otherContentType: true
 spec:
   version: 0.1
   type:
-    text: Documentation
+    text: Other content type
```

Comment:
@Hyperkid123 This line is the other value that I'm not sure how to update. I don't know where to check for a list of accepted values.

Reply:
Any string is accepted. Anything that can narrow down the document type would be helpful.